src/components/Divider/divider.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import Divider from './index'
+
+export default {
+  title: 'Data Display/Divider',
+  component: Divider,
+  parameters: {
+    componentSubtitle: 'Used to separate content.',
+  },
+}
+
+export const Horizontal = (): React.ReactElement => (
+  <>
+    <div>Some content</div>
+    <Divider />
+    <div>Some content2</div>
+  </>
+)
+
+export const Arrow = (): React.ReactElement => (
+  <>
+    <div>Some content</div>
+    <Divider withArrow />
+    <div>Some content2</div>
+  </>
+)
```

Comment:
Do we need the story?

Reply:
Yes, also we can review the set up of storybook in this repo to show the stories like we have in SRC https://components.gnosis-safe.io/